.mergify.yml

```diff
@@ -4,10 +4,18 @@ queue_rules:
       - or:
         - status-success=complete-pr
         - and:
-          # For more context, see "L10N - Auto Merge" rule down below
-          - base~=^releases[_/].*
-          - head~=^automation/sync-strings-\d+
+          # For more context, see "Auto Merge" rules down below
```

Comment:
Instead of making the "default" queue_rules more complex couldn't we add different dedicated queue rules for the l10n and ac-bump cases, and use them below in the corresponding actions?

Reply:
Great minds think alike! 😃 I thought about it too. Although, the docs reads multiple queues introduce a priority order[1]. At the moment, I don't believe we want to introduce such concept because it may confuse developers. E.g.: Why is this PR merged before mine? Why does the A-C/l10n bump keeps getting pushed at the bottom of the queue. 

To me at the moment, having a little bit of complexity in the current rule is a better tradeoff than inadvertently create some race conditions. 

I'm happy to revisit this, if this rule gets too big or if stakeholder believe a priority system is worth it.

[1] https://docs.mergify.com/actions/queue/#multiple-queues